lighthouse-core/test/audits/image-aspect-ratio-test.js

```diff
@@ -18,7 +18,7 @@ function generateRecord(url = 'https://google.com/logo.png', mimeType = 'image/p
 
 function generateImage(clientSize, naturalSize, networkRecord, props, src = 'https://google.com/logo.png') {
   Object.assign(networkRecord || {}, {url: src});
-  const image = {src, networkRecord};
+  const image = {src, ...networkRecord};
```

Comment:
wait, is this just for `resourceSize` and `mimeType`? :P

Reply:
removed